openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1505,6 +1505,40 @@ def _get_parameter_handler_class(self, tagname):
             raise KeyError(msg)
         return ph_class
 
+    def get_partial_charges(self, molecule, **kwargs):
+        """Generate the partial charges for the given molecule in this force field.
+
+        As any `ParameterHandler` may in principle modify charges, the entire
+        force field must be applied to the molecule to produce the charges.
+        Calls to this method from incorrectly or incompletely specified `ForceField`
+        objects thus may raise an exception.
+
```

Comment:
from @Yoshanuikabundi Add a docstring example of using this to ASSIGN partial charges to a mol.

Reply:
Done in 43a27a6